tests/scf5/input.dat

```diff
@@ -168,6 +192,10 @@ E = energy('scf')
 compare_values(Eref_rohf_can, E, 6, 'Triplet Disk CUHF energy') #TEST
 clean()
 
-set scf_type df
+set scf_type disk_df
+E = energy('scf')
+compare_values(Eref_rohf_df, E, 6, 'Triplet Disk DF CUHF energy') #TEST
+
+set scf_type mem_df
 E = energy('scf')
-compare_values(Eref_rohf_df, E, 6, 'Triplet DF CUHF energy') #TEST
+compare_values(Eref_rohf_df, E, 6, 'Triplet Mem DF CUHF energy') #TEST
```

Comment:
if you have time, would you edit the tests/pytests/test_addons.py accordingly?

then I'll expand stdsuite ref testing.